### ISSUE TITLE
No features in bestand-veehouderijbedrijven.geojson for province of Groningen?

### ISSUE THREAD

**Issue by FrieseWoudloper** [FarmHackNL/data#4]
It seems to me there are no features located in the province of Groningen, Fryslân or Drenthe in bestand-veehouderijbedrijven.geojson. Am I right?

**Comment by ndkv** [comment]
Yes, that's correct. This dataset is created by Utrecht province, see https://data.overheid.nl/data/dataset/bestand-veehouderij-bedrijven

Do you know whether other provinces have similar datasets? We can maybe stitch them together and create a single one?
